sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/implementation/KubernetesClusterNetworkProfileImpl.java

```diff
@@ -50,8 +50,8 @@ public KubernetesClusterNetworkProfileImpl withDnsServiceIP(String dnsServiceIP)
     }
 
     @Override
+    @Deprecated
     public KubernetesClusterNetworkProfileImpl withDockerBridgeCidr(String dockerBridgeCidr) {
-        ensureNetworkProfile().withDockerBridgeCidr(dockerBridgeCidr);
         return this;
```

Comment:
Seems manual code here? You could put manual code part in a separate commit.

Reply:
There should be a getter as well?

Reply:
`KubernetesClusterNetworkProfileImpl` is an interface implementation class that must implement all methods in the interface, even if the method is marked as "deprecated".

Reply:
Strangely there's no corresponding getter method defined in the interface for this property. Guess we're good.

Reply:
I got it. The getter is `ContainerServiceNetworkProfile networkProfile()`